hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCleaner.java

```diff
@@ -106,12 +106,7 @@ public static void main(String[] args) {
 
     String dirName = new Path(cfg.basePath).getName();
     JavaSparkContext jssc = UtilHelpers.buildSparkContext("hoodie-cleaner-" + dirName, cfg.sparkMaster);
-    try {
-      new HoodieCleaner(cfg, jssc).run();
-    } catch (Throwable throwable) {
-      LOG.error("Fail to run cleaning for " + cfg.basePath, throwable);
-    } finally {
-      jssc.stop();
```

Comment:
could we just call `System.exit(1)` or a non-zero exit code. I think we can keep the spark context stopping that way and still meet the airflow issue @haggy brought up?

Reply:
@haggy : Can you make this change and update the PR ?

Reply:
Sure that sounds good. Sorry for the delay getting back to this PR 😅